task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -222,7 +222,7 @@ spec:
               any(.packages[]; .type == "rpm")
             else
               false
-            end' <<<"$1"
+            end' <<<"$1" 2>/dev/null 1>&2
```

Comment:
I don't think this is desirable for one reason - we're sending any valuable output to the sink, so if an error occurs (see https://github.com/konflux-ci/build-definitions/pull/1935), we wouldn't see anything in the logs. Errors in logs are paramount to investigation.
Note that the choice of redirecting the output in `function is_json` is also deliberate because we don't care about any errors, the sole purpose of that function is to figure out if input is JSON or not, no more semantics needed.

Reply:
Good point, let's redirect only stdout not stderr (or not use `-e` and change the `if` condition)